R/pkg/R/functions.R

```diff
@@ -1250,7 +1250,7 @@ setMethod("upper",
 #'}
 setMethod("var",
           signature(x = "Column"),
-          function(x, y = NULL, na.rm = FALSE, use) {
+          function(x) {
```

Comment:
Are these signature changes just for our specific implementation of this generic ? i.e. the `na.rm=F` will still work for base R function but just not for SparkR ?